src/App/Options/Cmd/SqsToKafka.hs

```diff
@@ -64,34 +65,35 @@ instance RunApplication CmdSqsToKafka where
 
     runConduit $
       receiveMessageC sqsUrl
-      .| effectC (handleMessage sr kafkaTopic producer)
-      .| ackMessageC sqsUrl
+      .| batchByOrFlush (BatchSize 10)
+      .| effectC (handleMessages sr kafkaTopic producer)
+      .| ackMessages sqsUrl
       .| sinkNull
     return ()
 
-receiveMessageC :: MonadAWS m => String -> Source m Message
+receiveMessageC :: MonadAWS m => String -> Source m (Maybe Message)
 receiveMessageC sqsUrl = do
   let rm = receiveMessage (T.pack sqsUrl)
   rmr <- send (rm & (rmMaxNumberOfMessages .~ Just 10))
-  forM_ (rmr ^.. rmrsMessages . each) Conduit.yield
+  forM_ (rmr ^.. rmrsMessages . each) $ Conduit.yield . Just
   receiveMessageC sqsUrl
```

Comment:
Do we at some point yield `Nothing`?  I think it is needed to ensure partial batches happen.

Reply:
Can `yield Nothing` when `rmr` has no messages.